lint-install.go

```diff
@@ -167,7 +168,13 @@ func updateFile(root string, basename string, content []byte, dryRun bool) (stri
 	change := gotextdiff.ToUnified(basename, basename, string(existing), edits)
 
 	if !dryRun {
-		if err := os.WriteFile(dest, content, 0o600); err != nil {
+		var err error
+		if content == nil && f != nil {
+			err = os.Remove(dest)
```

Comment:
I don't think this `os.Remove` call should be necessary - unless the user doesn't have write access to it.